packages/web-components/src/checkbox/README.md

```diff
@@ -0,0 +1,121 @@
+# Checkbox
+
+> An implementation of a [checkbox](https://w3c.github.io/html-reference/input.checkbox.html) as a form-connected web-component.
+
+<br />
+
+## **Design Spec**
+
+[Link to Checkbox Design Spec in Figma](https://www.figma.com/file/1a1hBVizk7aLH76IvrnPFs/Checkbox?node-id=1666-534&t=Y5ISi4tWguXRFMdP-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 Checkbox extends from the [FAST Checkbox](https://explore.fast.design/components/fast-checkbox) and is intended to be as close to the Fluent UI React 9 Checkbox implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
+
+<br />
+
+## Class: `Checkbox`
+
+<br />
+
+### **Component Name**
+
+`<fluent-checkbox>`
+
+<br />
+
+### **Component Template**
+
+FAST `checkboxTemplate`
+
+<br />
+
+### **Variables**
+
+| Name                    | Description          | Type                                         |
+| ----------------------- | -------------------- | -------------------------------------------- |
+| `checkboxLabelPosition` | label position types | `{ before: "before", after: "after" }`       |
```

Comment:
What are these variables? I cannot find them anywhere in code.

Reply:
These are the exported options from `checkbox.options.ts`